library/src/main/java/com/google/android/exoplayer2/extractor/flv/AudioTagPayloadReader.java

```diff
@@ -58,15 +63,26 @@ public void seek() {
   protected boolean parseHeader(ParsableByteArray data) throws UnsupportedFormatException {
     if (!hasParsedAudioDataHeader) {
       int header = data.readUnsignedByte();
-      int audioFormat = (header >> 4) & 0x0F;
+      audioFormat = (header >> 4) & 0x0F;
       int sampleRateIndex = (header >> 2) & 0x03;
+      int encodingSize = header & 0x01;
       if (sampleRateIndex < 0 || sampleRateIndex >= AUDIO_SAMPLING_RATE_TABLE.length) {
```

Comment:
Hah at this check by the way. sampleRateIndex is guaranteed to be in the range [0-3] because of the & 0x03 on where it's calculated. The values in AUDIO_SAMPLING_RATE_TABLE aren't ever used :). I'm going to push a change that cleans some of this up a little!